### PR TITLE
[#44] react and dejs examples not working

### LINKED ISSUE #44: /react and /dejs examples are broken
# Issue

### How to reproduce

1. Update deno to 1.2.1.
2. Run `deno run -A /examples/dejs/index.ts` or `deno run -A /examples/react/server.ts`.

```bash
error: TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname
                 ~~~
    at https://deno.land/std@v0.51.0/path/win32.ts:911:18

TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname;
                 ~~~
    at https://deno.land/std@v0.51.0/path/posix.ts:433:18

Found 2 errors.
```

## Details

Updating dejs to 0.8.0 fixes the issue.

```bash
$ deno --version
deno 1.2.1
v8 8.5.216
typescript 3.9.2
```

### Sidenote

External dependencies in /examples should be consolidated either with an importMap or using a `./examples/deps.ts` file, see https://github.com/asos-craigmorten/opine/issues/41.

### PR DESCRIPTION
**Pull request by FaberVitale**
# Issue

[#44](https://github.com/asos-craigmorten/opine/issues/44)

## Details

Fixes #44

* Resolves #44 and updates the following deps:
  - [dejs](https://deno.land/x/dejs) **0.70 -> 0.8.0** in `examples/`
  - [evt](https://deno.land/x/evt)  **1.7.13 -> 1.8.0**
  - [http_errors](https://deno.land/x/http_errors) **2.0.0 -> 2.10**

* These updates have the benefit of removing unversioned `deno.land/std` dependencies.

* set deno_version in `test.yml` to `1.2.1`

* ran deno fmt; we should exclude `docs/*`  path.

I've looked into an importmap solution but I don't think support from the vscode extension is good enough 
and importmap feature seems quite limiting in ts current implementation.

## CheckList

- [x] PR starts with [#44].
- [x] Has been tested (where required) before merge to main.

### CODE REVIEW

**Comment by asos-craigmorten** [comment]
Looking good.

I think importmap is still unstable and not 100% convinced whether it’s here to stay yet. We could consider `deps.ts`...

Think there are a few other references to Deno 1.2.0 that are likely worth upgrading to latest as well.

deno fmt suggestion sounds good.

**Comment by FaberVitale** [comment]
> Looking good.
> 
> I think importmap is still unstable and not 100% convinced whether it’s here to stay yet. We could consider `deps.ts`...

We'd force users to download react, react-dom, dejs, swagger etc... just to test a json response or a redirect:
Imo, It wouldn't be the best 1st impression.

**Comment by asos-craigmorten** [comment]
> We'd force users to download react, react-dom, dejs, swagger etc... just to test a json response or a redirect:
> Imo, It wouldn't be the best 1st impression.

Agree

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ doc:
 	@deno doc ./mod.ts
 
 fmt:
-	@deno fmt
+	@deno fmt ./src ./benchmarks/ ./test ./examples deps.ts mod.ts version.ts
 
 fmt-check:
 	@deno fmt --check
```
Comment (asos-craigmorten) [inline Makefile:32]: Need to update the check as well otherwise will fail because we haven’t formatted x, y and z

**Pull request merged**